tests/browser/plugins/adaptors/basic.js

```diff
@@ -903,4 +903,37 @@ export default function() {
 
 	});
 
+	test( `children of computations are also adapted (#3130)`, t => {
+		const arr = [ 'hello' ];
+		let count = 0;
+
+		const Adaptor = {
+			filter ( v ) { return typeof v === 'string'; },
+			wrap ( ractive, v ) {
+				return {
+					get () { return v + ' adapted'; },
+					teardown () { count++; }
+				};
+			}
+		};
+
+		const r = new Ractive({
+			target: fixture,
+			template: '{{#each list}}{{.}}{{/each}}',
+			adapt: [ Adaptor ],
+			computed: {
+				list: {
+					get () { return arr; },
```

Comment:
I also find this a strange case, where a computed gets data outside the instance. If the data was set on the instance, they'd probably have adapted values by then.

But then again, I'm not familiar with computation internals. 😁

Reply:
Just a dumb case to fake a computation :smile:. It doesn't matter _how_ it's computed for the test, just _that_ it's computed.